src/registration.cpp

```diff
@@ -301,6 +350,34 @@ void RegistrationImpl::getPointXYZRGB (const Frame* undistorted, const Frame* re
   }
 }
 
+void Registration::getPointXYZ(const Frame *undistorted, int r, int c, float &x, float &y, float &z) const
+{
+    impl_->getPointXYZ(undistorted,r,c,x,y,z);
+}
+
```

Comment:
I think it can be implemented like this:
```cpp
void Registration::getPointXYZ (const Frame* undistorted, int r, int c, float& x, float& y, float& z) const
{
  float dummy;
  impl_->getPointXYZRGB(undistorted, undistorted, r, c, x, y, z, dummy);
}
```
What do you think?